app/services/batched_remove_status_service.rb

```diff
@@ -50,6 +57,22 @@ def batch_stream_entries(statuses)
     end
   end
 
+  def batch_activity_json(account, statuses)
+    account.followers.inboxes.each do |inbox_url|
```

Comment:
Delete isn't an addressed activity, and publicInbox is only for activities that are addressed to the Public collection. this feels like it should maybe be addressed by https://github.com/w3c/activitypub/issues/242 although that hasn't been spec'd yet.